doc/api/globals.md

```diff
@@ -28,6 +28,19 @@ added: v0.1.103
 
 Used to handle binary data. See the [buffer section][].
 
+## Class: WebAssembly
+<!-- YAML
+added: v8.0.0
+-->
+
+<!-- type=global -->
+
+* {Function}
+
+The object that acts as the namespace for all W3C
+[Web Assembly][webassembly-org] related functionality. See the
```

Comment:
Micro-nit: Should it be `WebAssembly` rather than `Web Assembly`? I don't see the latter appear anywhere in the link provided. We should generally style things the way the authority providing them styles them, so I think that means _WebAssembly_ without a space here? But it's hardly the end of the world if anyone disagrees vehemently.

Reply:
Agreed and fixed!